pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -998,12 +998,16 @@ def __init__(self, edb_padstackinstance, _pedb):
         self._position = []
         self._pdef = None
 
-    @property
-    def terminal(self):
+    @pyaedt_function_handler
+    def get_terminal(self, name=None, create_new_terminal=False):
         """Return PadstackInstanceTerminal object."""
```

Comment:
```suggestion
        """Get padstack terminal."""
```
Add "Parameters" and "Returns" docstring sections.